react/src/components/ResourceGroupSelect.tsx

```diff
@@ -142,13 +139,13 @@ const ResourceGroupSelect: React.FC<ResourceGroupSelectProps> = ({
       {...searchProps}
       defaultValue={autoSelectDefault ? autoSelectedOption : undefined}
       onDropdownVisibleChange={(open) => {
-        if (open) {
-          startLoadingTransition(() => {
-            updateFetchKey();
-          });
-        }
+        // if (open) {
+        //   startLoadingTransition(() => {
+        //     updateFetchKey();
+        //   });
+        // }
```

Comment:
Is there a reason you commented this out?

Reply:
The `ResourceGroup` does not change frequently. It is not necessary to refetch it every time the dropdown is opened.